src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task {
+    protected String completeBy;
+//    protected boolean isDone;
```

Comment:
Avoid leaving commented code in the repo.
You can always recover these code using version control, and leaving it in may accumulate into a bigger problem later.